tracer/build/PackageVersionsGeneratorDefinitions.json

```
@@ -597,5 +597,23 @@
       "1.0.1",
       "1.*.*"
     ]
+  },
+  {
+    "IntegrationName": "Yarp",
+    "SampleProjectName": "Samples.Yarp.DistributedTracing",
+    "NugetPackageSearchName": "Yarp.ReverseProxy",
+    "MinVersion": "1.0.0",
+    "MaxVersionExclusive": "2.2.0",
+    "SpecificVersions": [
+      "1.1.2",
+      "2.*.*"
+    ],
+    "VersionConditions": [{
+      "MaxVersionExclusive": "2.0.0"
+    }, {
```

I don't think we want this do we? 🤔 Otherwise we shouldn't test 2.x versions at all IIUC. Though, as we are, I don't really understand 😂 

```suggestion
    "VersionConditions": [{
```

---

I think you're right that I need to remove this. My goal is that for 1.x package versions we test against all target frameworks listed in the csproj and for 2.x versions we only test against net6.0+ . I'll rework this in a new commit

---

Fixed the versions in b59e6ef55c3e420920cc987aaa958666caffcb4b, I think